__tests__/app.test.js

```diff
@@ -113,20 +113,28 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  it("200: responds with an empty array if passed an existing topic that has no articles", () => {
+    return request(app)
+      .get("/api/articles?topic=paper")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toEqual([]);
+      });
+  });
   it("400: responds with the correct error message when passed an invalid topic query", () => {
     return request(app)
       .get("/api/articles?topic=123")
```

Comment:
I think we can remove this test. If you check my comment in your model you'll see my logic as to why!